### PR TITLE
Declare KodiWrapper locally when using reuselanguageinvoker

### PR DESCRIPTION
**Pull request by mediaminister**
This makes sure all `KodiWrapper` functions are executed instantaneous and return fresh values when using `reuselanguageinvoker`

This fixes https://github.com/pietje666/plugin.video.vrt.nu/issues/560

This makes realtime toggling the "watching activity" setting possible. (Should also work for all other settings)

### CODE REVIEW

File: resources/lib/addon.py [hunk 1]
```diff
@@ -15,131 +15,130 @@
...
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_continue_menu(page=1)
+    VRTPlayer(KodiWrapper(globals())).show_continue_menu(page=1)
 
 
 @plugin.route('/resumepoints/refresh')
 def resumepoints_refresh():
     ''' The API interface to refresh the resumepoints cache '''
     from resumepoints import ResumePoints
-    ResumePoints(kodi).refresh(ttl=0)
```
Comment (michaelarnauts) [inline resources/lib/addon.py]: You will only need to do this once per function.

**Comment by michaelarnauts** [comment]
I've had a similar issue in VTM GO, since the Kodi wrapper is actually only initialised once, the old settings that are read in the first __init__ call are being used. Therefore, you need to read the setting every time instead of using the cached value.

**Comment by michaelarnauts** [comment]
Hmm, just looked at it, and this doesn't seem to be the case. Ignore my comment :)

File: resources/lib/addon.py (hunk 1 above)
Comment (mediaminister) [reply resources/lib/addon.py]: Indeed, I did a quick find and replace. Fixed it.

**Comment by mediaminister** [comment]
Thanks for reporting. If VTM GO add-on is not affected, there might be a cleaner fix for this problem.

**Comment by dagwieers** [comment]
If we would instantiate in addon_entry.py and then pass it on, would that not help?

Personally I would use a global variable rather than a KodiWrapper object that we have to pass on.
The object is always needed anyway, so I don't see a good reason not to define it as global.

And then kodiwrapper.py would just hold a list of functions (something that would be more portable than what we currently have).

**Comment by dagwieers** [comment]
@mediaminister The actual fix is in #575, the problem is that the settings seems to be read only when Addon() is instantiated, which means you don't see any updates to the settings for a reused Addon() instance.

**Pull request closed without merging**